### PR TITLE
refactor: remove legacy Enter keyCode

### PR DESCRIPTION
**Pull request by wsuwt**
## Description

Remove legacy Enter `keyCode`, I'm not sure if we do need Enter for `toggle` though. Checkbox and radio buttons which are in the same family do not support `Enter`.

## Type of change

Please delete options that are not relevant.

- [x] Refactor (improves code without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/toggle/index.ts [hunk 1]
```diff
@@ -132,9 +132,6 @@ export class Toggle extends ControlElement {
     if (event.key === 'Enter' || event.key === ' ' || event.key === 'Spacebar') {
       this.handleCheckedChange();
     }
-    else if (event.keyCode && event.keyCode === 13 || event.keyCode === 32) { // For older browsers
```
Comment (Sakchai-Refinitiv) [inline packages/elements/src/toggle/index.ts:135]: We still support IE 11 in v6, why did you remove this?

Comment (wsuwt) [reply packages/elements/src/toggle/index.ts:135]: You're right. That means Enter event listeners in `multi-input` and `slider` should use `event.keyCode` too.  I will repurpose this PR for adding legacy Enter support.

Comment (Sakchai-Refinitiv) [reply packages/elements/src/toggle/index.ts:135]: I think the `event.key` works fine on IE11, have you tested it yet?

Comment (Sakchai-Refinitiv) [reply packages/elements/src/toggle/index.ts:135]: It works

**Review by Sakchai-Refinitiv — APPROVED** [review]
LGTM
